core/build.gradle

```diff
@@ -71,6 +71,7 @@ sourceSets {
             include "io/crate/core/StringUtils*"
             include "io/crate/geo/GeoJSONUtils*"
             include "io/crate/types/**"
+            include "io/crate/exceptions/*"
```

Comment:
why is this necessary? We're only returning SQLActionException to the client and don't want to have the other exception classes on the client, as otherwise we'd break compatibility more often